node/rpcstack.go

```diff
@@ -52,9 +53,9 @@ type HTTPServer struct {
 	WsOrigins          []string
 	Timeouts           rpc.HTTPTimeouts
 
-	RPCAllowed bool
-	WSAllowed  bool // TODO discuss this later bc possible race condition
-	GQLAllowed bool
+	RPCAllowed *atomic.Value
```

Comment:
this will be changed to int32

use atomic.storeint32 when storing new val and just use LoadInt32 when checking val